etc/vault_secret.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+LOCAL_OVERRIDE_FILE=./etc/vault_secret.local.sh
+
+if [[ ! -z "${ANSIBLE_VAULT_PASSWORD}" ]]; then
```

Comment:
I wonder if this should automatically detect being run on the login servers, and load it from a standardised location?

Or actually, given for now this will only get run on login, should we just set `vault_password_file` to wherever we store it?

Reply:
> I wonder if this should automatically detect being run on the login servers, and load it from a standardised location?

Probably yes.

> Or actually, given for now this will only get run on login, should we just set vault_password_file to wherever we store it?

Definitely not. No way to do local dev if we do that. My `local.sh` calls out to my password manager to fetch it.